moto/iam/responses.py

```diff
@@ -20,6 +20,20 @@ def detach_role_policy(self):
         template = self.response_template(GENERIC_EMPTY_TEMPLATE)
         return template.render(name="DetachRolePolicyResponse")
 
+    def attach_group_policy(self):
```

Comment:
If you rebase your work on the latest Moto master branch you'll see that these already exist. Great minds think alike! :)

Try this:
```
git remote set-url origin https://github.com/spulec/moto.git
git remote set-url josenrihernand git@github.com:josenrihernand/moto.git
git fetch origin
git rebase origin/master
# handle merge conflicts
git push -f josenrihernand HEAD:IAM_get_policy
```